src/main/java/de/blackforestsolutions/datamodel/CallStatus.java

```diff
@@ -1,20 +1,38 @@
 package de.blackforestsolutions.datamodel;
 
+import com.fasterxml.jackson.core.JsonProcessingException;
+import de.blackforestsolutions.datamodel.util.LocoJsonMapper;
+import lombok.AccessLevel;
 import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
 import org.springframework.lang.NonNull;
 
+@Slf4j
 @Getter
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
 public class CallStatus<T> {
 
-    private final T calledObject;
+    private T calledObject;
 
-    private final Status status;
+    private Status status;
```

Comment:
why are they not final anymore?

Reply:
Because we have a private noargsconstructor annotated in the beginning of the class with lombok, was necessary due to jackson mapper.

Reply:
As we have only our allArgsConstructor public all fields are set automatically. So it makes no different anyway.